.github/workflows/sycl-linux-precommit.yml

```diff
@@ -154,6 +154,21 @@ jobs:
             extra_lit_opts: --param matrix-xmx8=True --param gpu-intel-dg2=True
             env: '{"LIT_FILTER":${{ needs.determine_arc_tests.outputs.arc_tests }} }'
             use_igc_dev: true
+          - name: E2E tests on Intel Ponte Vecchio GPU
+            runner: '["Linux", "pvc"]'
+            env: '{"LIT_FILTER_OUT":"ESIMD/unified_memory_api/"}'
+            image: ghcr.io/intel/llvm/ubuntu2404_intel_drivers:latest
+            image_options: -u 1001 --device=/dev/dri --privileged --cap-add SYS_ADMIN
+            target_devices: ext_oneapi_level_zero:gpu;opencl:gpu
+            extra_lit_opts: -j 50
+          - name: E2E tests with dev igc on Intel Ponte Vecchio GPU
+            runner: '["Linux", "pvc"]'
+            env: '{"LIT_FILTER_OUT":"ESIMD/unified_memory_api/"}'
+            image: ghcr.io/intel/llvm/ubuntu2404_intel_drivers:devigc
+            image_options: -u 1001 --device=/dev/dri --privileged --cap-add SYS_ADMIN
+            target_devices: ext_oneapi_level_zero:gpu;opencl:gpu
```

Comment:
nit: i think we can remove `ext_oneapi_` and just have `level_zero`

Reply:
IMO, there should be only one [non-deprecated] spelling and we should be using that. I vaguely remember trying to do that (including discussion what it should be in different places, e.g. `sycl-ls` output), but maybe I failed to bring it to completeness.

Reply:
agree, and i propose we should use `level_zero` as the single option, as it's shorter and what's used in `sycl-ls` today:
```
[level_zero:gpu][level_zero:0] Intel(R) oneAPI Unified Runtime over Level-Zero, Intel(R) Iris(R) Xe Graphics 12.0.0 [1.6.31740+14]
```

Reply:
A bit of history: https://github.com/intel/llvm/pull/12185 (and links from it). IMO, we should also prohibit `ext_oneapi_Level_zero` in the `lit.cfg.py` now (by removing extra support/hack for it).

Reply:
I also see `ext_oneapi_hip` and `ext_oneapi_cuda` in workflow files (https://github.com/intel/llvm/blob/sycl/.github/workflows/sycl-post-commit.yml#L61C29-L61C43). I think we also have to replace them with `hip` and `cuda` respectively before we can drop support for `ext_oneapi_` in LIT (https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/lit.cfg.py#L476)

Reply:
PR to drop support for `ext_oneapi_` in LIT: https://github.com/intel/llvm/pull/16508